modules/calib3d/src/calibinit.cpp

```diff
@@ -471,6 +471,110 @@ static void icvBinarizationHistogramBased(Mat & img)
     }
 }
 
+static bool findCornerNeighbor(
+    cv::AutoBuffer<ChessBoardQuad>& all_quads,
+    const std::vector<Point2f>& all_quads_pts,
+    flann::GenericIndex<flann::L2_Simple<float>>& all_quads_pts_index,
+    const int idx,
+    const ChessBoardQuad& cur_quad,
+    const int i,
+    const cv::Point2f& pt,
+    float& min_dist,
+    const float radius,
+    std::vector<int>& neighbors_indices,
+    std::vector<float>& neighbors_dists,
+    const float thresh_scale,
+    int& closest_quad_idx,
+    ChessBoardQuad*& closest_quad,
+    int& closest_corner_idx,
+    cv::Point2f& closest_corner_pt)
+{
+    if (cur_quad.neighbors[i])
```

Comment:
The condition should be moved outside the function. `i` is not used inside the function.

Reply:
Done